docs/react-native/v2/how-to-guides/extend-capabilities/noise-cancellation.mdx

```diff
@@ -21,7 +21,11 @@ The Noise Cancellation feature employs a sophisticated AI model trained specific
 
 `@100mslive/react-native-hms` version 1.10.2 or later is required to utilize the Noise Cancellation feature in your React Native application.
 
-Also, this feature has gated access currently. To enable Noise Cancellation in your rooms, reach out to **support@100ms.live** or connect with us on [100ms Discord](https://discord.com/invite/kGdmszyzq2).
+<Note type='warning'>
+**IMPORTANT** <br/>
+Enable Noise Cancellation in the template configuration. Learn more on enabling this feature from [here](/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation)
```

Comment:
Sentence update:

Learn more about enabling this feature [here](https://github.com/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation).